includes/wikia/services/sass/Filter/InlineImageFilter.class.php

```diff
@@ -52,6 +52,8 @@ protected function encodeFile( $fileName ) {
 		$parts = explode('.', $fileName);
 		$ext = end($parts);
 
+		$base64 = true;
```

Comment:
I know that we have here base64 encoding as default and we turn it off for svg... But on other hand there isn't default encoding, but encoding for each image type...
I think we could choose more descriptive name for the $base64 variable (eg: $enable_base64)...